assets/js/sticky-menu.js

```diff
@@ -147,6 +148,11 @@ define( ['jquery', 'underscore'], function ( $, _ ) {
 		/**
 		 * Is the direction of the scroll = up?
```

Comment:
@krumpak this comment is not representing the below method, so update it accordingly.